src/atomate2/vasp/flows/phonons.py

```diff
@@ -0,0 +1,326 @@
+    effects, a correction of the dynamical matrix based on BORN charges can
+    be performed.     Finally, phonon densities of states, phonon band structures
+    and thermodynamic properties are computed.
+
+    .. Note::
+        It is heavily recommended to symmetrize the structure before passing it to
+        this flow. Otherwise, a different space group might be detected and too
+        many displacement calculations will be generated.
+        It is recommended to check the convergence parameters here and
+        adjust them if necessary. The default might not be strict enough
+        for your specific case.
+
+    Parameters
+    ----------
+    name : str
+        Name of the flows produced by this maker.
+    sym_reduce : bool
+        Whether to reduce the number of deformations using symmetry.
+    symprec : float
+        Symmetry precision to use in the
+        reduction of symmetry to find the primitive/conventional cell
+        (use_primitive_standard_structure, use_conventional_standard_structure)
+        and to handle all symmetry-related tasks in phonopy
+    displacement: float
+        displacement distance for phonons
+    min_length: float
+        min length of the supercell that will be build
+    prefer_90_degrees: bool
+        if set to True, supercell algorithm will first try to find a supercell
+        with 3 90 degree angles
+    get_supercell_kwargs: additional arguments
+        to determine supercell
+    use_primitive_standard_structure: bool
+        this will enforce to start the phonon computation
+        from the primitive standard structure
+        according to Setyawan, W., & Curtarolo, S. (2010).
+        High-throughput electronic band structure calculations:
+        Challenges and tools. Computational Materials Science,
+        49(2), 299-312. doi:10.1016/j.commatsci.2010.05.010.
+        This makes it possible to use certain k-path definitions
+        with this workflow. Otherwise, we must rely on seekpath
+    use_conventional_standard_structure: bool
+        this will enforce to start the phonon computation
+        from the conventional standard structure
+        according to Setyawan, W., & Curtarolo, S. (2010).
+        High-throughput electronic band structure calculations:
+        Challenges and tools. Computational Materials Science,
+        49(2), 299-312. doi:10.1016/j.commatsci.2010.05.010.
+        We will however use seekpath and primitive structures
+        as determined by from phonopy to compute the phonon band structure
+    bulk_relax_maker : .BaseVaspMaker or None
+        A maker to perform a tight relaxation on the bulk.
+        Set to ``None`` to skip the
+        bulk relaxation
+    static_energy_maker : .BaseVaspMaker or None
+        A maker to perform the computation of the DFT energy on the bulk.
+        Set to ``None`` to skip the
+        static energy computation
+    generate_phonon_diplsacements_kwargs: dict
```

Comment:
```suggestion
    generate_phonon_displacements_kwargs: dict
```

Reply:
Will remove this and the next kwargs as they are not further used in the code.